src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -342,6 +352,14 @@ In fact, [image CDNs](/image-cdns/) in particular are a great choice because the
 While image CDNs are a great way to reduce resource load times, using a third-party domain to host your images comes with an additional connection cost. While preconnecting to the origin can mitigate some of this cost, the best option is to serve images from the same origin as your HTML document. Many CDNs allow you to proxy requests from your origin to theirs, which is a great option to look into if available.
 {% endAside %}
 
+#### Reduce contention for network bandwidth
+
+Even if you've reduced the size of your resource and the distance it has to travel, a resource can still take a long time to load if you're loading many other resources at the same time. This problem is know as _network contention_.
+
+If you've given your resource a [high `fetchpriority`](/priority-hints/) and [started loading it as soon as possible](#1.-eliminate-resource-load-delay) then the browser will do its best to prevent lower-priority resources from competing with the LCP resource. However, if you're loading many resources with high `fetchpriority`, or if you're just loading a lot of resources, then there's only so much the browser can do.
+
+In general, you only want to give a high `fetchpriority` to resources that are needed for the initial render (like stylesheets) or the one or two resources most likely to be your LCP resource.
```

Comment:
Not sure I like the implication here that you should explicitly use `fetchpriority` on stylesheets? It’s implicit (assuming you’re loading in standard way) so I’d rather avoid suggesting to use them more than necessary - the very point of this line!

Reply:
Yeah, I didn't intend to imply that someone would put `fetchpriority` on a stylesheet (though I agree that's what it sounds like from the way I worded it). I'll rework.